music21/roman.py

```diff
@@ -832,6 +859,11 @@ def romanNumeralFromChord(chordObj,
         'i64b3': 'Sw43',
         'io6b5b3': 'Ger65',
     }
+    secondaryDominantSwaps = {
+        ('#', 'i'): ('vii', 'ii'),
```

Comment:
I'd prefer not to have this one, since it's not a secondary  dominant -- would need some sort of "preferAppliedChords" instead of secondary dominant.

Reply:
Ok. I think we'll want that other functionality, but I'll leave it out for now. I'm also including a suggested rename of `preferSecondaryDominants` to `preferSecondaryV` to this effect.